kubetest/e2e.go

```diff
@@ -495,7 +495,7 @@ func perfTest() error {
 
 func chartsTest() error {
 	// Run helm tests.
-	if err := finishRunning(exec.Command("/src/k8s.io/charts/test/helm-test-e2e.sh")); err != nil {
+	if err := finishRunning(exec.Command("%s/src/k8s.io/charts/test/helm-test-e2e.sh", os.Getenv("GOPATH"))); err != nil {
```

Comment:
Is $GOPATH set automatically to `/go/`?

Reply:
yeah, from base image that's `/go`, you can also add a customized $GOPATH from prow config